### ISSUE TITLE
PyPi pip install aiven-client appears to be broken

### ISSUE THREAD

**Issue by allen-munsch** [aiven/aiven-client#128]
Does not work: `python3 -m pip install aiven-client`

```
05:40:53 (venv_ok) jmunsch@pop-os heroku-buildpack-aiven-deploy ±|master ✗|→ python3 -m pip install aiven-client
Collecting aiven-client
  Using cached https://files.pythonhosted.org/packages/30/f0/67d40e7a2091a95e72dec23d6627a994c8111869e8f732a2d8fc2ca0e2e2/aiven-client-2.3.2.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /home/jmunsch/PycharmProjects/propertymeld/venv_ok/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-i2r96qi3/aiven-client/setup.py'"'"'; __file__='"'"'/tmp/pip-install-i2r96qi3/aiven-client/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: /tmp/pip-install-i2r96qi3/aiven-client/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-i2r96qi3/aiven-client/setup.py", line 45, in <module>
        version=version.get_project_version("aiven/client/version.py"),
      File "/tmp/pip-install-i2r96qi3/aiven-client/version.py", line 34, in get_project_version
        % version_file)
    Exception: version not available from git or from file '/tmp/pip-install-i2r96qi3/aiven-client/aiven/client/version.py'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
WARNING: You are using pip version 19.2.3, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

These all fail with the same error:

```shell
2301  08/01/20 17:45:48 python3 -m pip install https://github.com/aiven/aiven-client/archive/2.2.3.tar.gz
 2302  08/01/20 17:47:32 python3 -m pip install https://github.com/aiven/aiven-client/archive/2.3.2.zip
 2303  08/01/20 17:47:50 python3 -m pip install https://github.com/aiven/aiven-client/archive/2.3.1.zip
 2304  08/01/20 17:47:58 python3 -m pip install https://github.com/aiven/aiven-client/archive/2.3.0.zip
 2305  08/01/20 17:48:07 python3 -m pip install https://github.com/aiven/aiven-client/archive/2.2.5.zip
 2306  08/01/20 17:48:19 python3 -m pip install https://github.com/aiven/aiven-client/archive/2.2.4.zip
```

Works: `python3 -m pip install git+https://github.com/aiven/aiven-client.git`

```
05:50:37 (venv_ok) jmunsch@pop-os propertymeld ±|APP-520-jm-1 ✗|→ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Pop!_OS 18.04 LTS
Release:	18.04
Codename:	bionic

05:52:56 (venv_ok) jmunsch@pop-os propertymeld ±|APP-520-jm-1 ✗|→ python3 --version
Python 3.6.7
```

**Comment by rikonen** [comment]
Thank you for the report. https://github.com/aiven/aiven-client/pull/129 should fix this. We'll release new version as soon as that has been merged

**Comment by rikonen** [comment]
Should be fixed in 2.3.3
